sycl/doc/EnvironmentVariables.md

```diff
@@ -11,7 +11,7 @@ subject to change. Do not rely on these variables in production code.
 | Environment variable | Values | Description |
 | -------------------- | ------ | ----------- |
 | SYCL_PI_TRACE | Any(\*) | Force tracing of PI calls to stderr. |
-| SYCL_BE | PI_OPENCL, PI_OTHER | When SYCL RT is built with PI this controls which plugin to use. Default value is PI_OPENCL. |
+| SYCL_BE | PI_OPENCL, PI_CUDA, PI_OTHER | When SYCL RT is built with PI, this controls which plugin to use. Default value is PI_OPENCL. |
```

Comment:
```suggestion
| SYCL_BE | PI_OPENCL, PI_CUDA, PI_OTHER | When SYCL RT is built with PI, this controls which plugin is used by the `default_selector`. Default value is PI_OPENCL. |
```

Reply:
As far as I understand all plugins are still available irrespective of this variable, and one can pick any device (OpenCL or CUDA) with a dedicated device selector; only the default device selector is affected by `SYCL_BE`.

Reply:
Thanks for noting this, I've applied the suggestion